src/app/components/pie-chart/pie-chart.css

```
@@ -22,6 +24,42 @@ mat-card {
+  padding: 20px !important;
+}
+
+/* Style for chart labels to improve readability */
+::ng-deep .pie-label {
+  font-size: 12px !important;
+  fill: #000000 !important;
+  font-weight: bold;
+}
+
+/* Style for tooltips with increased padding */
+::ng-deep .ngx-charts-tooltip {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: ng-deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)